src/app/components/icons/index.tsx

```diff
@@ -36,3 +36,17 @@ export const RightChevron = ({ className }: { className?: string }) => (
     <path d="M21.6 14.3L5.5 31h6.4l14.6-15L11.9 1H5.5l16.1 16.7v-3.4z" />
   </svg>
 );
+
+export const CloseButton = () => (
+  <svg
+    viewBox="0 0 32 32"
+    focusable="false"
+    aria-hidden="true"
+    width="17"
+    height="17"
+    stroke="white"
```

Comment:
Heya, I'm not sure if setting the stroke colour here to white is the best approach. I notice with other SVGs we have applied colour via CSS.

I raise this as I checked the banner on High Contrast mode and no-css and the white colour if hard to see on both.

![Screenshot 2023-10-26 at 09 08 42](https://github.com/bbc/simorgh/assets/98817636/f5ebc2fa-6241-4481-ae69-b8d45c34113f)
![Screenshot 2023-10-26 at 09 11 23](https://github.com/bbc/simorgh/assets/98817636/4b9cb628-5e01-451b-b27f-57ec13aa2c1a)

Reply:
Heya on this I've made a suggestion in the PR - https://github.com/bbc/simorgh/pull/11158. 
(Apologies I don't know how to do a commit suggestion/ a better way to show my thoughts!)

Happy to talk it through. It's using a psuesdo 'after' element to apply the hover/ focus, which is what we use on the Nav bar and the media player.